force-app/main/default/classes/CRLP_ResetRollupFieldsQueueable_TEST.cls

```diff
@@ -134,4 +138,44 @@ private class CRLP_ResetRollupFieldsQueueable_TEST {
         System.assertEquals(false, updatedAccounts[0].CustomizableRollups_UseSkewMode__c,
             'Customizable Rollups Use Skew Mode flag should remain false');
     }
+
+    /**
+     * @description Create Rollup__mdt records to test rolling up from the Opp and Payment objects to the Account
+     */
+    private static void mockRollupCMTValues() {
+
+        // Create a single Filter Group (no need for filter rules here)
+        String filterGroupId1 = CMT_UnitTestData_TEST.getNewRecordId();
+        String filterGroupId2 = CMT_UnitTestData_TEST.getNewRecordId();
+
+        // Extremely simple Filter Rule for IsWon only
+        String filterGroupsJSON = '[' +
+            CMT_UnitTestData_TEST.createFilterGroupRecord(filterGroupId1, 'TestFilterGroup1-IsWon+Paid') + ',' +
+            CMT_UnitTestData_TEST.createFilterGroupRecord(filterGroupId2, 'TestFilterGroup2-IsWon+WrittenOff') +
+            ']';
+        String filterRulesJSON = '[' +
+            CMT_UnitTestData_TEST.createFilterRuleRecord(null, filterGroupId1, 'Group1.Rule1', 'Opportunity', 'IsWon', 'Equals', 'True') + ',' +
+            CMT_UnitTestData_TEST.createFilterRuleRecord(null, filterGroupId1, 'Group1.Rule2', 'npe01__OppPayment__c', 'npe01__Paid__c', 'Equals', 'True') + ',' +
```

Comment:
The filter rules against the Payment Object aren't necessary here. In fact, you could just have zero filter groups/rules for the purposes of this test class.

Reply:
Cool.  I'll pull them.  Thanks!!

Reply:
Rules and filters have been removed.  Also switched to run in parallel.